packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewer.js

```diff
@@ -204,7 +204,7 @@ class LightCurveViewer extends Component {
     if (!this.isCurrentTaskValidForAnnotation()) return // Sanity check
 
     const annotations = this.annotationBrushes
-      .filter((raw) => (raw.minX !== undefined && raw.maxX !== undefined))
+      .filter((raw) => (raw.minX !== undefined && raw.maxX !== undefined && (raw.maxX - raw.minX) > 0))
```

Comment:
I think this is the wrong place to check for this. Other kinds of data might have valid negative x values and negative values would be valid for the x center that we record in the annotation. I think we want the final value of the width to be greater than zero, so I think we can achieve that by getting its absolute value. 

Hypothetically if negative values are valid for the data, let's say minX is -8 and maxX is -6, center x would be -7 and the width as it is currently calculated would be -2 which doesn't make sense, but the absolute value of 2 does.

Reply:
I'm very unsure of this, but as is, if minX is -8 and maxX is -6 would width be 2 (-6 - -8)? The part I'm not sure about is how raw.minX/raw.maxX (i.e. d3?) would interpret mins and maxs in negative value data territory.

Reply:
Regardless, I think the checks for `null` in the filter, and adding`Math.abs` in width declaration makes more sense (recent commit). Thanks!

Reply:
Sorry yes, my logic is backwards with the min and max (I think I typed out too quickly!), but we can get a negative number if max is -8 and min is -6, so I think taking the absolute value is the right direction. Thanks!

I do think we have some possible issues with d3, but we're short circuiting it by adding a check before adding the annotation to the classification. Investigating what d3 is doing here is a follow up.